mu-plugins/global-fonts/index.php

```diff
@@ -141,44 +141,52 @@ function get_font_url( $font, $subset ) {
 	}
 
 	switch ( $lower_font ) {
+		case 'courier prime':
+			$font_folder    = 'Courier Prime/';
+			$font_file_name = 'CourierPrime-Regular';
+			break;
+		case 'courier prime bold':
+			$font_folder    = 'Courier Prime/';
```

Comment:
I'm not sure if the space matters here: `Courier Prime/`. The other two are either hyphenated or the space is removed..?

Reply:
I imagine spaceless folder names are safer across file systems.

Reply:
Ah good catch, don't think I properly registered what this was